stix2/test/test_campaign.py

```diff
@@ -23,8 +23,8 @@ def test_campaign_example():
     campaign = stix2.Campaign(
         id="campaign--8e2e2d2b-17d4-4cbf-938f-98ee46b3cd3f",
         created_by_ref="identity--f431f809-377b-45e0-aa1c-6a4751cae5ff",
-        created="2016-04-06T20:03:00Z",
-        modified="2016-04-06T20:03:00Z",
+        created="2016-04-06T20:03:00.000Z",
```

Comment:
Just to be clear, we shouldn't *need* to pass in the microsecond precision for the constructor args, correct? I'm fine doing it in most cases, but there should be at least one test to confirm it's not required.